docs/manuals/core/config/scenario-config.md

```diff
@@ -80,19 +83,21 @@ from taipy import Config
 # 2: {0: -7, 1: -4, 2: 0, 3: -8},
 # 3: {0: 1, 1: 4, 2: 8, 3: 0}}
 def compare_function(*data_node_results):
-    comparison_result= {}
-    current_result_index = 0
-    for current_result in data_node_results:
-        comparison_result[current_result_index]={}
-        next_result_index = 0
-        for next_result in data_node_results:
-            print(f"comparing result {current_result_index} with result {next_result_index}")
-            comparison_result[current_result_index][next_result_index] = next_result - current_result
-            next_result_index += 1
-        current_result_index += 1
-    return comparison_result
-
-scenario_config = Config.configure_scenario("multiply_scenario", [pipeline_config], comparators={datanode_config.id: compare_function})
+    compare_result= {}
+    current_res_index = 0
+    for current_res in data_node_results:
+        compare_result[current_res_index]={}
```

Comment:
Please choose whether to use spaces around the equal sign (as on next) line, or not.